templates/overrides/search/views-view-unformatted.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file
+ * Theme implementation to display showcase_search unformatted rows.
+ *
+ * @see ./core/modules/views/templates/views-view-unformatted.html.
+ */
+#}
+{% if view.current_display == 'showcase_search_page' %}
+  {% if title %}
+    <h3>{{ title }}</h3>
+  {% endif %}
+  {% for row in rows %}
+    {% set row_classes = [
+      default_row_class ? 'views-row',
+    ] %}
+    <div{{ row.attributes.addClass(row_classes) }}>
+      {{- row.content -}}
+    </div>
+  {% endfor %}
+{% endif %}
+<hr class="d-none d-md-block mt-4-5">
```

Comment:
remove this file, the pager should receive this hr, this is for 3 good reasons:
1. the line should not appear if no pager is there
2. we cant target showcase_search_page
3. this hr should be consistent in all listings where a pager is needed